templates/serverpod_templates/projectname_server_upgrade/config/development.yaml

```diff
@@ -48,3 +48,9 @@ maxRequestSize: 524288 # The maximum size of requests allowed in bytes
 sessionLogs:
   persistentEnabled: true
   consoleEnabled: true
+
+# futureCallExecutionEnabled: true # Defaults to true
+
+#futureCall:
+#  concurrencyLimit: 1 # Defaults to 1, a negative or null value removes the limit
+#  scanInterval: 5000 # Defaults to 5000
```

Comment:
@SandPod Sorry late to the party and the ship has probably already sailed.

But am I right in assuming that this would be milliseconds?

Personally I would opt to include the unit here (else many will just add a comment themselves).